src/components/AdminBoard.vue

```diff
@@ -48,5 +55,13 @@ export default {
     .admin-button-bottom {
       grid-area: bottom;
     }
+
+    .admin-text {
+      grid-area: text;
+      text-align: center;
+      writing-mode: vertical-rl;
+      text-orientation: sideways-right;
+      font-size: constants.$admin-board-width;
```

Comment:
So I like that the admin board is 30px wide, but setting the font like this makes the admin-board 32+ pixels wide. I don't see a problem with hardcoding the font size, or you can subtract some from the current value.